gridcomps/Cap/FlapCLI.F90

```diff
@@ -7,42 +7,30 @@ module MAPL_FlapCLIMod
    use FLAP
    use mapl_KeywordEnforcerMod
    use mapl_ExceptionHandling
-   use mapl_CapOptionsMod, only:  MAPL_CapOptions !Rename is for backward compatibility. Remove renaming for 3.0
+   use mapl_CapOptionsMod, only:  MAPL_CapOptions
    implicit none
    private
 
    public :: MAPL_FlapCLI
-   public :: MAPL_CapOptions !Needed for backward compatibility. Remove for 3.0
 
-   type :: MAPL_FlapCLI
+   type :: MAPL_FlapCLI_Type
      type(command_line_interface) :: cli_options
    contains
       procedure, nopass :: add_command_line_options
       procedure :: fill_cap_options
-   end type MAPL_FlapCLI
-
-   interface MAPL_FlapCLI
-      module procedure new_CapOptions_from_flap
-      module procedure new_CapOptions_from_flap_back_comp
-   end interface MAPL_FlapCLI
-
-   interface MAPL_CapOptions !Needed for backward compatibility. Remove for 3.0
-      module procedure old_CapOptions_from_flap
-   end interface MAPL_CapOptions
-
+   end type MAPL_FlapCLI_Type
 
 contains
 
-   function new_CapOptions_from_flap(unusable, description, authors, dummy, rc) result (cap_options)
+   function MAPL_FlapCLI(unusable, description, authors, rc) result (cap_options)
```

Comment:
```suggestion
   function FlapCLI(unusable, description, authors, rc) result (cap_options)
```

Reply:
Should I bring the interface back so that external components call MAPL_FlapCLI?

Reply:
Let's discuss in teams.    I just want to avoid the heavy use of `MAPL_` as a prefix on objects and procedures.    These are great/essential for languages that lack namespace management, but ... Fortran has had it for 31 years now.   :-)